### PR TITLE
Custom javascript operators and actions, as well as header and body html.

### PR DESCRIPTION
**Pull request by Gervwyk**

### CODE REVIEW

File: packages/servers/serverDev/src/shell/public/modules/filterDefaultValue.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (SamTolmay) [inline packages/servers/serverDev/src/shell/public/modules/filterDefaultValue.js:1]: Remove this file

File: packages/servers/serverNetlify/src/shell/index.html [hunk 2]
```diff
@@ -20,15 +20,39 @@
       <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (SamTolmay) [inline packages/servers/serverNetlify/src/shell/index.html:20]: Remove head.html and body.html

File: packages/docs/actions/JsAction.yaml [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+      ```
+      (params: {
+        name: string,
+        args?: any[]
+      }): void
+      ```
+    description: |
+      The `JsAction` action is used to call a custom JavaScript function which was loaded onto the page using the `window.lowdefy.registerJsAction()`. This JavaScript function can be asynchronous. See [Custom JavaScript](/custom-javascript) for more details on how to register a new JavaScript action.
+
+      The returned result of the JavaScript function is accessible through the [`_actions`](/_actions) operator for subsequent action in the event action chain.
```
Comment (SamTolmay) [inline packages/docs/actions/JsAction.yaml:31]: subsequent actions

**Comment by Gervwyk** [comment]
Happy with the changes. We can merge the PR. I'm blocked to approve since it is my PR.

**Review by SamTolmay — APPROVED** [review]

**Comment by SamTolmay** [comment]
Closes #585

**Pull request merged**